web/templates/message_feed_errors.hbs

```diff
@@ -20,10 +20,17 @@
         {{/tr}}
         &nbsp;
         <span>
+            {{#if is_guest}}
+            {{#tr}}
+            Consider <z-link>searching all public channels that you can view</z-link>.
+            {{#*inline "z-link"}}<a class="search-shared-history" href="">{{> @partial-block}}</a>{{/inline}}
+            {{/tr}}
+            {{else}}
             {{#tr}}
             Consider <z-link>searching all public channels</z-link>.
             {{#*inline "z-link"}}<a class="search-shared-history" href="">{{> @partial-block}}</a>{{/inline}}
             {{/tr}}
+            {{/if}}
         </span>
```

Comment:
Should we indent the `tr` tags to make the nesting more clear?

Reply:
Done